### PR TITLE
storage: Make it appear as an Application

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
Should this have an icon?

File: pkg/Makefile.am [hunk 1]
```diff
@@ -26,6 +26,7 @@ metainfo_DATA = pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml
 		pkg/kdump/org.cockpit-project.cockpit-kdump.metainfo.xml \
 		pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml \
 		pkg/machines/org.cockpit-project.cockpit-machines.metainfo.xml \
+		pkg/storaged/org.cockpit-project.cockpit-storaged.metainfo.xml \
```
Comment (martinpitt) [inline pkg/Makefile.am]: Can you please rm the new file in the "when not building optional packages, remove their files" section, too?

You can test with tools/make-rpm when setting `build_optional` to 0 in the spec. I bet it currently fails with an "this file has not been packaged anywhere *mimimi*"

**Comment by martinpitt** [comment]
Forgive me for cancelling semaphore, we have some contention there and this will need an update anyway.

**Comment by martinpitt** [comment]
Ah, and the file must also be added to tools/debian/cockpit-storage.install, see [failure](https://209.132.184.41:8493/logs/pull-11780-20190509-091313-c4be5389-verify-debian-stable/log.html)

**Comment by mvollmer** [comment]
Thanks, fixed!

**Pull request merged**